UPGRADING.md

```diff
@@ -2,6 +2,14 @@
 
 This document describes breaking changes and how to upgrade. For a complete list of changes including minor and patch releases, please refer to the [changelog](CHANGELOG.md).
 
+## v4
+
+Upgraded to `readable-stream@3` which contain several substantial changes and improvements. Since `level-iterator-stream` derives from `readable-stream` we decided to bump major as well. See https://github.com/nodejs/readable-stream#version-3xx for more details.
```

Comment:
```suggestion
Upgraded to [`readable-stream@3`](https://github.com/nodejs/readable-stream#version-3xx) which contains several substantial changes and improvements. Since `level-iterator-stream` derives from `readable-stream` we decided to bump major as well.
```